doc/ale-scala.txt

```diff
@@ -2,6 +2,37 @@
 ALE Scala Integration                                       *ale-scala-options*
 
 
+===============================================================================
+scalafmt                                                   *ale-scala-scalafmt*
+
+If Nailgun is used, override `g:ale_scala_scalafmt_executable` like so: >
+  let g:ale_scala_scalafmt_executable = 'ng'
+
+To disable `scalafmt` globally, use |g:ale_fixers| like so: >
+  let g:ale_fixers = {'scala': []}
```

Comment:
The documentation isn't quite right here. Fixers are never enabled by default, so there's no need to disable them. You can remove this part of the documentation..

Reply:
Thanks @w0rp , I've removed those bits.